librz/util/utype.c

```diff
@@ -397,6 +397,19 @@ RZ_API int rz_type_unlink(Sdb *TDB, ut64 addr) {
 	return true;
 }
 
+static bool sdbdeletelink(void *p, const char *k, const char *v) {
```

Comment:
this whole utype file is IMO out of place in RzUtil :/

Reply:
I will address this during `RzTypes` refactoring, forget about this for now.